### DISCUSSION TITLE
剧集分组的参数列表在哪可以看到全部呀

### DISCUSSION THREAD

**Discussion by youyou-sudo** [ikaros-dev/ikaros#616] [Q&A] [answered]
剧集分组的参数列表在哪可以看到呀，在使用 API 写一个播放页，目前只知道 `SPECIAL_PROMOTION` 和 `MAIN` 两个参数，分别对应的是 `特典(SP)`  和 `正片` ，如果有一个表就好了

![image](https://github.com/ikaros-dev/ikaros/assets/60727068/1c41261b-73f1-4de3-9d83-4f03aec860c6)

**Comment by chivehao** [comment, marked as answer]
看这个：<https://github.com/ikaros-dev/ikaros/blob/master/api/src/main/java/run/ikaros/api/store/enums/EpisodeGroup.java>
